pubspec.yaml

```diff
@@ -75,6 +75,8 @@ dependencies:
   audio_video_progress_bar: ^0.10.0
   model_viewer_plus: 1.1.2
 
+  flutter_image_compress: ^1.1.0
```

Comment:
How much it effects the size of the app?

Reply:
will check into it

Reply:
What was the result?

Reply:
29.35 MB with flutter_image_compress, 29.30 MB without flutter_image_compress